runironic.sh

```diff
@@ -26,6 +26,7 @@ mkdir -p /shared/log/ironic
 /usr/bin/ironic-api --log-file  /shared/log/ironic/ironic-api.log &
 
 /bin/runhealthcheck "ironic" &>/dev/null &
+/bin/runexporterapp &
```

Comment:
Let's not re-add this as well.

Reply:
and this, otherwise how do we run the app ?